src/components/Header/Header.tsx

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, {
+  memo, useEffect, useRef, useState,
+} from 'react';
+import { Todo } from '../../types/Todo';
+
+export type Props = {
+  onSubmit: (todo: Omit<Todo, 'id'>) => void
+  showErrorMessage: (v: string) => void
+  userId: number
+  temporaryTodo: Todo | null
+};
+
+export const Header: React.FC<Props> = memo(({
+  onSubmit,
+  showErrorMessage,
+  userId,
```

Comment:
You don't need to pass this value, just pass it in `onSubmit function` in App component